src/health_checker.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+#ifndef HEALTH_CHECKER_H_
+#define HEALTH_CHECKER_H_
+
+#include <boost/algorithm/string/join.hpp>
```

Comment:
Do all of these need to be `#include`d in the header? Can we move some to the `.cc` file?

Reply:
Moved.